### PR TITLE
Change "lincense" -> "license" and add an article

### PR DESCRIPTION
**Pull request by laurenz**

### CODE REVIEW

**Review by susanmdouglas-aws — APPROVED** [review]
It looks good to me...

**Review by nasbyj — DISMISSED** [review]

**Review by 3manuek — DISMISSED** [review]

File: _docs/contributing.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ nav_order: 2
 ## Contributing code
 
 Babelfish is available as Open Source software and can be downloaded from Github
```
Comment (rcv-aws) [inline _docs/contributing.md:9]: The product name is 'Babelfish for PostgreSQL'

**Review by rcv-aws — CHANGES_REQUESTED** [review]
use correct product name

**Comment by laurenz** [comment]
Thanks for the review. I pushed a commit that is more specific about this.

**Review by 3manuek — APPROVED** [review]

**Comment by laurenz** [comment]
Please approve and merge this.

**Review by rcv-aws — APPROVED** [review]

**Pull request merged**